### PR TITLE
Moving to typescript pt 3

### PR DESCRIPTION
**Pull request by mariabug**

### CODE REVIEW

**Comment by mariabug** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #20** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/20" target="_blank">Graphite</a> 
    * **PR #21** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/21" target="_blank">Graphite</a> 
      * **PR #22** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/22" target="_blank">Graphite</a> 
        * **PR #23** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/23" target="_blank">Graphite</a> 
          * **PR #24** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/24" target="_blank">Graphite</a>  👈
            * **PR #25** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/25" target="_blank">Graphite</a> 
              * **PR #26** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/26" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DonArtisan/technify-react/24?utm_source=stack-comment).

**Comment by mariabug** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #20** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/20" target="_blank">Graphite</a> 
    * **PR #21** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/21" target="_blank">Graphite</a> 
      * **PR #22** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/22" target="_blank">Graphite</a> 
        * **PR #23** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/23" target="_blank">Graphite</a> 
          * **PR #24** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/24" target="_blank">Graphite</a>  👈
            * **PR #25** <a href="https://app.graphite.dev/github/pr/DonArtisan/technify-react/25" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DonArtisan/technify-react/24?utm_source=stack-comment).

File: src/pages/account/Account.tsx [hunk 1]
```diff
@@ -128,6 +146,7 @@ export function Account() {
                     <Form>
                       <ModalBody>
                         <FastField name="firstName">
+                          {/* @ts-expect-error */}
```
Comment (AguilarKevin) [inline src/pages/account/Account.tsx:149]: Formik momento

File: src/pages/products/index.tsx [hunk 2]
```diff
@@ -30,7 +32,7 @@ export default function Products() {
       paddingX={{md: 20, lg: 52}}
     >
       {products?.edges.map(({node: product}, index) => (
-        <ProductCard key={index} product={product} />
+        <ProductCard key={index} product={product as Product} />
```
Comment (AguilarKevin) [inline src/pages/products/index.tsx]: lo mismo del pr pasado xd

**Review by AguilarKevin — APPROVED** [review]

**Pull request merged**